Patches/Rimsenal Collection/Federation/ThingDefs_Misc/Ammo_Fed.xml

```diff
@@ -47,7 +47,7 @@
 		</graphicData>		
 		<projectile Class="CombatExtended.ProjectilePropertiesCE">
 			<damageDef>Plasma</damageDef>
-			<speed>50</speed>
+			<speed>0</speed>
```

Comment:
Won't this throw errors?

Reply:
Not if this is a projectile for a mortar-like weapon.